specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/Registry/Archive.tsp

```diff
@@ -137,9 +90,10 @@ interface Archives {
   /**
    * Lists all archives for the specified container registry and package type.
    */
-  @action("archives")
-  @list
-  list is AchieveOps.Read<Archive, Response = ArmResponse<ArchiveListResult>>;
+  list is AchieveOperationOps.List<
+    Archive,
+    Response = ArmResponse<ArchiveListResult>
+  >;
```

Comment:
two issues:
this is a LIST operation of an ARM resource (list itself by its parent), therefore we should use `List` template instead of `Read` template.
`Read` template always represents to the `GET` method of an ARM resource, it does not represent "any arbitrary method with the verb GET". For instance List is also Get.